docs/json_schemas/dns_record/v0/provider.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$defs": {
+    "BaseModel": {
+      "properties": {},
+      "title": "BaseModel",
+      "type": "object"
+    },
+    "DNSRecordProvider": {
+      "description": "List statuses for the DNS records informed by the requirer.",
+      "properties": {
+        "dns_domains": {
+          "default": null,
+          "description": "List statuses for the domains requested by the requirer.",
+          "items": {
+            "$ref": "#/$defs/DnsProviderData"
+          },
+          "title": "Dns Domains",
+          "type": "array"
+        },
+        "dns_entries": {
+          "default": null,
+          "description": "List of statuses for the DNS records requested by the requirer.",
+          "items": {
+            "$ref": "#/$defs/DnsProviderData"
+          },
+          "title": "Dns Entries",
+          "type": "array"
+        }
+      },
+      "title": "DNSRecordProvider",
+      "type": "object"
+    },
+    "DnsProviderData": {
+      "properties": {
+        "uuid": {
+          "description": "UUID for this domain as specified by the requirer.",
+          "examples": "550e8400-e29b-41d4-a716-446655440000",
+          "minLength": 1,
+          "name": "UUID",
+          "title": "Uuid",
+          "type": "string"
+        },
+        "status": {
+          "allOf": [
+            {
+              "$ref": "#/$defs/Status"
+            }
+          ],
+          "description": "Status for the domain request.",
+          "examples": [
+            "approved",
+            "invalid_credentials"
+          ],
+          "name": "Status"
+        },
+        "status_description": {
+          "default": null,
+          "description": "Status description.",
+          "examples": [
+            "incorrect username and password"
+          ],
+          "name": "Status description",
+          "title": "Status Description",
+          "type": "string"
+        }
+      },
+      "required": [
+        "uuid",
+        "status"
+      ],
+      "title": "DnsProviderData",
+      "type": "object"
+    },
+    "Status": {
+      "description": "Represent the status values.",
+      "enum": [
+        "approved",
+        "invalid_credentials",
+        "permission_denied",
+        "conflict",
+        "validation",
```

Comment:
I think it's a bit ambiguous. We should probably use "validation_error" or "invalid_data"